.github/workflows/just-nut.yml

```diff
@@ -66,14 +65,12 @@ jobs:
         with:
           node-version: lts/*
           cache: yarn
-      # TODO: Remove sfdx-cli install after parity
-      # many setup commands require sfdx executable
-      - name: Install CLIs
+      - name: Install CLI
```

Comment:
this is correct for v2, but should go into main (just-nuts on main can use both clis)